### PR TITLE
DOC Document lazy loading

### PR DESCRIPTION
**Pull request by GuySartorelli**
Replaces https://github.com/silverstripe/silverstripe-framework/pull/10361
Adds docs for https://github.com/silverstripe/silverstripe-admin/issues/1308

## Issue
- https://github.com/silverstripe/developer-docs/issues/426

### CODE REVIEW

File: en/02_Developer_Guides/15_Customising_the_Admin_Interface/02_CMS_Architecture.md [hunk 1]
```diff
@@ -774,6 +774,26 @@ The URL endpoints `{$AdminURL}mytabs/tab1` and `{$AdminURL}mytabs/tab2`
 should return HTML fragments suitable for inserting into the content area,
 through the `PjaxResponseNegotiator` class (see above).
 
+### Lazy loading fields on tab activation
+
+When a tab is not lazy loaded via ajax, it might still be necessary to
+delay some work (for example when doing HTTP requests) until the tab is activated. This is how, for example, the
+[`GridFieldLazyLoader`](api:SilverStripe\Forms\GridField\GridFieldLazyLoader) works.
+
+In order to open up the same kind of features to other fields, a custom event is fired on all nodes with the `lazy-loadable` class inside the activated tab panel.
```
Comment (sabina-talipova) [inline en/02_Developer_Guides/15_Customising_the_Admin_Interface/02_CMS_Architecture.md:783]: Looks good. Small question, should user make additional steps to add `lazy-loadable` class to the field, e.g. `$fields->addExtraClass('lazy-loadable')`?
I think, probably would be nice to provide full example how to do this to be more clear.

Comment (ChloeSartorelli) [reply en/02_Developer_Guides/15_Customising_the_Admin_Interface/02_CMS_Architecture.md:783]: Yeah good point, will add.

Comment (ChloeSartorelli) [reply en/02_Developer_Guides/15_Customising_the_Admin_Interface/02_CMS_Architecture.md:783]: Done

**Review by sabina-talipova — APPROVED** [review]
LGTM!

**Pull request merged**